guides/common/modules/proc_installing-the-discovery-service.adoc

```diff
@@ -9,33 +9,29 @@ Complete the following procedure to enable the Discovery service on {SmartProxyS
 ifdef::satellite,orcharhino[]
 [options="nowrap" subs="+quotes,attributes"]
 ----
-# {package-install-project} foreman-discovery-image tfm-rubygem-smart_proxy_discovery
+# {foreman-installer} \
+--enable-foreman-proxy-plugin-discovery \
+--foreman-proxy-plugin-discovery-install-images=false
 ----
 endif::[]
-ifdef::foreman-el,katello[]
+ifndef::satellite,orcharhino[]
 [options="nowrap" subs="+quotes,attributes"]
 ----
 # {foreman-installer} \
 --enable-foreman-proxy-plugin-discovery \
---foreman-proxy-plugin-discovery-install-images=true \
 --foreman-proxy-plugin-discovery-source-url=http://downloads.theforeman.org/discovery/releases/3.5/
```

Comment:
Please keep `--foreman-proxy-plugin-discovery-install-images=true` and remove `--foreman-proxy-plugin-discovery-source-url=http://downloads.theforeman.org/discovery/releases/3.5/`

Reply:
Addressed :+1: